Signal/src/Models/TSMessageAdapaters/TSVideoAttachmentAdapter.m

```diff
@@ -151,6 +151,7 @@ - (UIView *)mediaView {
         _audioPlayPauseButton = [[UIButton alloc] initWithFrame:CGRectMake(3, 3, 30, 30)];
         [_audioPlayPauseButton setBackgroundImage:[UIImage imageNamed:@"audio_play_button"]
                                          forState:UIControlStateNormal];
+        _audioPlayPauseButton.enabled = NO;
```

Comment:
This button should never be enabled since it's just visual decoration.  Playback is handled through MessageViewController's "message tapped" logic.